core/src/io/anuke/mindustry/ui/fragments/BlockConfigFragment.java

```diff
@@ -79,8 +79,10 @@ public boolean hasConfigMouse(){
         return e != null && (e == table || e.isDescendantOf(table));
     }
 
+    private static final Runnable empty = () -> {};
```

Comment:
This is unnecessary. First of all, you can do `e.update(null)`, and second of all, there's no need for the elements to stop updating at all. When a table is not visible, its elements do not update.

Reply:
If the block config UI is opened, and then it is deconstructed, the game will crash because of `NullPointerException`.
For some reason, after deconstructing it, the update function is called once more, and when `tile` is accessed in the update function, and the block is being deconstructed so `tile` is `null`, `NullPointerException` is raised.

And for the part about providing `null` instead of `() -> {}`, I'll look into it.